src/main/webapp/aboutus.html

```diff
@@ -18,247 +18,297 @@
 <html>
```

Comment:
The large diff is due to whitespace changes, change to 2 spaces to see what's actually changed.